proto/one/operations/sample/analyte_results_import_file.proto

```diff
@@ -0,0 +1,35 @@
+syntax = "proto3";
+
+import "analyte_results_import_telemetry.proto";
+
+option csharp_namespace ="ONE.Models.CSharp";
+
+message AnalyteResultsImportFile {
+    
+    // Path to look for files ("/sample-data").
+    string path = 1;
+
+    // File name pattern ("*.csv").
+    string fileName = 2;
+
+    // Field delimiter (",").
+    string delimiter = 3;
+
+    // Ignore blank lines or not.
+    bool ignoreBlankLines = 4;
+
+    // Indicates if the file has header row(s).
+    bool hasHeaderRow = 5;
+
+    // Number of rows to skip at the start of the file to account for the header.
+    int32 rowsToSkip = 6;
+
+    // Specifies the format of DateTime data ("yyyy-MM-ddTHH:mm:ss").
+    string dateTimeColumnFormat = 7;
+
+    AnalyteResultsImportTelemetries telemetry = 8;
```

Comment:
Would be better if telemetry is in plural. E.g. Telemetries.

Reply:
This is how the structure was defined here: https://aquaticinformatics.atlassian.net/wiki/spaces/OP/pages/422183066/Schema+Configuration+Data+Analysis+Results+Import+Profile

Any thoughts @JonathanRobertson-AQI ?

Reply:
@KentMiller , I am okay to with the same name, however, just wanna bring the point out since it is not like exisitng pattern, and protobuf change usually is breaking change when it is shiping out or persisted somewhere.

Reply:
Thanks @AlanLiu-AI. I appreciate the feedback and don't have an issue with changing the name if that's what the team wants, just want to make sure everyone is aligned before making the change. This is part of an epic that is due to go through ADR and would be good to discuss there.